views/pages/organisations/index.html

```diff
@@ -0,0 +1,19 @@
+{% extends "details-template.njk" %}
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set govukServiceNameKey = translate("govuk.serviceName") %}
+{% set hmpoPageKey = "user-management.organisationsPage" %}
+{% set govukServiceUrl = "/organisations" %}
+
+{% block mainContent %}
+    {{ super() }}
+
+{% if orgInfo.name == undefined %}
+<h2>Organisation does not exist</h2>
+{% endif %}
+
+<h2>{{ orgInfo.name }}</h2>
```

Comment:
Yeah just doesn't  appear - it's more placeholder content for now anyhow. 

Probably worth noting that the iam-api returns a 400 when the org doesn't exist and we intercept it as an error in the ui...if we wanted do, we could and assuming we could render an error page instead of conditional text if required...